WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingController.java

```diff
@@ -84,6 +82,7 @@
     private static final Logger _log = LogManager.getLogger(WNPRC_PurchasingController.class);
     private static final DefaultActionResolver _actionResolver = new DefaultActionResolver(WNPRC_PurchasingController.class);
     public static final String NAME = "wnprc_purchasing";
+    public static final Double OVER_FIVE_K = 5000.0;
```

Comment:
I'd probably call this TOTAL_REQUIRING_ADDITONAL_REVIEW or something. The constant itself isn't over 5K, after all, it's 5K exactly.

And since uses convert it to BigDecimal, why not make the constant a BigDecimal?

Reply:
true, done